app/assets/src/components/views/report/TaxonModal.jsx

```diff
@@ -18,15 +17,12 @@ class TaxonModal extends React.Component {
       wikiUrl: null
     };
 
-    this.loadedBackground = this.handleOpen = this.handleOpen.bind(this);
-    this.handleClose = () => this.setState({ open: false });
     this.linkTo = this.linkTo.bind(this);
   }
 
-  handleOpen() {
+  componentWillMount() {
```

Comment:
This function is being deprecated. You should use functions in the new react lifecycle. 
You can probably just use `componentDidMount`.